modules/data.atmosphere/R/met.process.R

```diff
@@ -53,8 +148,6 @@ met.process <- function(site, input_met, start_date, end_date, model, host, bety
       cmdFcn  = paste0(pkg,"::",fcn,"(",paste0("'",args,"'",collapse=","),")")
       remote.execute.R(cmdFcn,host$name,user=NA, verbose=TRUE)
       
-      #download.NARR(outfolder, start_date, end_date, raw.host=host$name, overwrite=FALSE)
-      
       mimetype =  'application/x-netcdf'
```

Comment:
These variables should be gotten from the remote call. This will return the right format type and formatname as well as the real path to the file on disk.